src/backend/distributed/planner/distributed_planner.c

```diff
@@ -2549,21 +2549,20 @@ HasUnresolvedExternParamsWalker(Node *expression, ParamListInfo boundParams)
 		/* check whether parameter is available (and valid) */
 		if (boundParams && paramId > 0 && paramId <= boundParams->numParams)
 		{
-			ParamExternData *externParam = NULL;
+			Oid paramType;
```

Comment:
just to avoid having an uninitialized variable fwiw

```suggestion
			Oid paramType = InvalidOid;
```